### PR TITLE
Feat forms ui

### PR DESCRIPTION
**Pull request by rishi9684**
#App dropdown added
#Admin app layout added
#navigation content added 
#grid templates added for invitee, seat maps, badge

### CODE REVIEW

File: config/templates/grids/invitetemplate-grid/body.vue [hunk 1]
```diff
@@ -169,8 +169,14 @@
 </template>
 
 <script>
+import nuxtconfig from '~/nuxt.config'
 export default {
   props: ['items'],
+  computed: {
+    imgurl() {
```
Comment (SukantGujar) [inline config/templates/grids/invitetemplate-grid/body.vue:176]: Call it cdnUrl only

Comment (SukantGujar) [reply config/templates/grids/invitetemplate-grid/body.vue:176]: In a component you don't need to import Nuxt config. It's already available as $config.

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: config/templates/grids/invitetemplate-grid/body.vue (hunk 1 above)
Comment (rishi9684) [reply config/templates/grids/invitetemplate-grid/body.vue:176]: ok

Comment (rishi9684) [reply config/templates/grids/invitetemplate-grid/body.vue:176]: cdnUrl  Done
#if i remove  import Nuxt config getting error

Comment (rishi9684) [reply config/templates/grids/invitetemplate-grid/body.vue]: added publicRuntimeConfig in nuxt config as discuss

**Review by bitpod-io — APPROVED** [review]

**Review by SukantGujar — APPROVED** [review]

File: config/templates/grids/invitetemplate-grid/body.vue [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                  :src="item.ImageURL"
+                  :lazy-src="item.ImageURL"
+                  aspect-ratio="1"
+                  class="grey lighten-2"
+                  min-height="200"
+                  max-height="200"
+                >
+                  <template v-slot:placeholder>
+                    <v-img
+                      src="https://res.cloudinary.com/mytestlogo/image/upload/v1571132783/invitee-image.png"
```
[the remark this answers is not in the document]
Comment (rishi9684) [reply config/templates/grids/invitetemplate-grid/body.vue]: added in config

**Pull request merged**